gateway-service/src/test/java/org/zowe/apiml/gateway/filters/pre/ServiceAuthenticationFilterTest.java

```diff
@@ -97,7 +97,7 @@ public void increment(String name) {
             assertTrue(zre.getCause() instanceof ZuulException);
             ZuulException ze = (ZuulException) zre.getCause();
             assertEquals(HttpStatus.INTERNAL_SERVER_ERROR.value(), ze.nStatusCode);
-            assertEquals(String.valueOf(new RuntimeException("Potential exception")), ze.errorCause);
+            assertEquals(new RuntimeException("Potential exception").getLocalizedMessage(), ze.errorCause);
```

Comment:
What is the reason? I guess this is an equivalent:
```suggestion
            assertEquals("Potential exception" ze.errorCause);
```